ui/apps/platform/src/Containers/Collections/CollectionsFormPage.tsx

```diff
@@ -75,7 +73,7 @@ function CollectionsFormPage({
     const [deleteId, setDeleteId] = useState<string | null>(null);
     const [modalCollectionId, setModalCollectionId] = useState<string | null>(null);
 
-    const [saveError, setSaveError] = useState<CollectionSaveError | undefined>();
+    const { saveError, setSaveError, onSubmit } = useCollectionFormSubmission(pageAction);
```

Comment:
First impression of `saveError` is as an imperative verb.

Although `errorFromSave` feels verbose, it might inspire you to see another alternative.

Reply:
I have briefly thought about renaming these `formSubmitError` and `setFormSubmitError` to match the `onSubmit` function. I'll push the change in a few.

Reply:
Hmm actually now that I look at this the change would be 60+ occurrences across 5 files. I'll save this as a follow up to avoid the extra overhead and testing in this PR.